src/modules/app/__tests__/pickList-test.js

```diff
@@ -52,9 +80,21 @@ describe('pickList', () => {
     const refreshMainObject$ = new Subject();
     const removeObject$ = testScheduler.createHotObservable(removeObject, {1: {id: 1, museumNo: 'H1'}});
     const refreshObject$ = testScheduler.createHotObservable(refreshObject,
-      {1: {id: 1456, objectId: 1, path: ',1,2,', pathNames: [{nodeId: 1, name: 'Test'}, {nodeId: 2, name: 'Tull'}]}});
+      {
+        1: {id: 1456, objectId: 1, path: ',1,2,', pathNames: [{nodeId: 1, name: 'Test'}, {nodeId: 2, name: 'Tull'}]},
+        2: {id: 14567, objectId: 2, path: ',1,4,', pathNames: [{nodeId: 1, name: 'Test'}, {nodeId: 4, name: 'Tull 4'}]}
+      }
+    );
     const clearObjects$ = testScheduler.createHotObservable(clearObjects);
-    const addObject$ = testScheduler.createHotObservable(addObject, {1: {value: {id: 1}, path: []}});
+    const addObject$ = testScheduler.createHotObservable(addObject, {1: {value: {id: 1}, path: []}, 2: {value: {id: 2}, path: []}});
+
+    const refreshObjects$ = testScheduler.createHotObservable(refreshObjects,
```

Comment:
Nice job. But lets think a little step longer. Lets also test the method on object model, getObjectLocations! :)